low-level/framework/rabbitmq/logging_processor.py

```diff
@@ -21,56 +21,51 @@
 """
 
 import json
-import time
 from syslog import (LOG_ALERT, LOG_CRIT, LOG_DEBUG, LOG_EMERG, LOG_ERR,
                     LOG_INFO, LOG_NOTICE, LOG_WARNING)
+import time
 
-import pika

```

Comment:
can we remove this pika dependency too, if no where else we are using

Reply:
Done